frontend/src/components/Share/Share.scss

```diff
@@ -16,3 +16,46 @@
   display: block;
   width: 25px;
 }
+
+.modal {
+    padding: 20px;
+    background: #fff;
+    border-radius: 4px;
+    width: 100%;
+    position: fixed;
+    bottom: 0;
+    left: 0px;
+    box-shadow: 0 0 10px rgba(0, 0, 0, 0.2);
+    text-align: center;
+}
+
+input#copy {
+    padding: 10px;
+    margin-left: 0;
+    margin-bottom: 10px;
+    border-radius: 10px;
```

Comment:
use the border-radius variable https://github.com/Covexit/covexit/blob/master/frontend/src/stylebase/variables.scss